e2e/igraphalg_test/test_leiden/test.yml

```diff
@@ -0,0 +1,19 @@
+query: >
+    CALL igraphalg.community_leiden() YIELD community_id, community_members
+    RETURN community_id, extract(node IN community_members | node.id) AS community_members
+    ORDER BY community_id ASC
+
+output:
+    - community_id: 0
+      community_members: [0]
+    - community_id: 1
+      community_members: [1]
+    - community_id: 2
+      community_members: [2]
+    - community_id: 3
+      community_members: [3]
+    - community_id: 4
+      community_members: [4]
+    - community_id: 5
+      community_members: [5]
+ 
```

Comment:
Try downloading the "new line" formatter in VSCode or whatever you are using. There won't be flags like these in the code then.